package.json

```diff
@@ -57,7 +57,7 @@
   "scripts": {
     "build": "gatsby build && npm run build-admin",
     "build-admin": "cp -r static/admin public/ && node ./build-admin.js",
-    "develop": "gatsby develop",
+    "develop": "gatsby develop -H 0.0.0.0",
```

Comment:
Hadn't meant to get this in this PR, but locally this makes us listen on all interfaces. That helps me test when I work in a VM.